cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -573,6 +573,10 @@ func (csr *ClusterStateRegistry) updateReadinessStats(currentTime time.Time) {
 			klog.Warningf("Failed to get nodegroup for %s: %v", unregistered.Node.Name, errNg)
 			continue
 		}
+		if nodeGroup == nil {
+			klog.Warningf("nodegroup is nil for %s", unregistered.Node.Name)
```

Comment:
Please make it uppercase :)

Reply:
> Please make it uppercase :)

Done in https://github.com/kubernetes/autoscaler/pull/2096/commits/91016a605af14635d8470ad031cbf41934cc2ae0